examples/pytorch/graphsaint/README.md

```diff
@@ -0,0 +1,118 @@
+# GraphSAINT
+
+This DGL example implements the sampling method proposed in the paper [GraphSAINT: Graph Sampling Based Inductive Learning Method](https://arxiv.org/abs/1907.04931).
+
+Paper link: https://arxiv.org/abs/1907.04931
+
+Author's code: https://github.com/GraphSAINT/GraphSAINT
+
+Contributor: Liu Tang ([@lt610](https://github.com/lt610))
+
+## Dependecies
+
+- Python 3.7.0
+- PyTorch 1.6.0
+- numpy 1.19.2
+- dgl 0.5.3
+
+## Dataset
+
+All datasets used are provided by Author's [code](https://github.com/GraphSAINT/GraphSAINT). They are available on [Google Drive link](https://drive.google.com/drive/folders/1zycmmDES39zVlbVCYs88JTJ1Wm5FbfLz) (alternatively, [BaiduYun link (code: f1ao)](https://pan.baidu.com/s/1SOb0SiSAXavwAcNqkttwcg#list/path=%2F)). Dataset summary("m" stands for multi-class classifification, and "s" for single-class.):
```

Comment:
"[BaiduYun link (code: f1ao)]" -> "[BaiduYun (code: f1ao)]". Also it seems that the official name is "Baidu Wangpan
" (https://en.wikipedia.org/wiki/Baidu_Wangpan)?